README.md

```diff
@@ -146,10 +145,7 @@ To run tests, run the following command:
 
```

Comment:
- [ ] Please kindly fix the MIT broken link in your README file, having broken links in the README file doesn’t look very professional 😎